src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import {blockNew} from '../utils/cn';
+
+import {AvatarIcon} from './AvatarIcon';
+import {AvatarImage} from './AvatarImage';
+import {AvatarText} from './AvatarText';
+import type {AvatarProps} from './types/main';
+
+import './Avatar.scss';
+
+const b = blockNew('avatar');
+
+export const Avatar = React.forwardRef<HTMLDivElement, AvatarProps>((props, ref) => {
+    const {
+        size = 'm',
+        theme = 'normal',
+        view = 'filled',
+        backgroundColor,
+        borderColor,
+        title,
+        ariaAttributes,
```

Comment:
The only role is suitable for this component is `role="img"`, therefore only applicable aria props are: `aria-label` and `aria-labelledby`. Let's specify them separately

Reply:
`none`, `presentation` and `generic` are applicable too. I suggest giving them the ability to change it.